packages/analytics-js-plugins/src/errorReporting/index.ts

```diff
@@ -38,21 +61,74 @@ const ErrorReporting = (): ExtensionPlugin => ({
       );
     },
     notify: (
-      pluginEngine: IPluginEngine,
-      client: any,
-      error: Error,
+      pluginEngine: IPluginEngine, // Only kept for backward compatibility
+      client: any, // Only kept for backward compatibility
+      error: SDKError,
       state: ApplicationState,
       logger?: ILogger,
+      httpClient?: IHttpClient,
+      errorState?: ErrorState,
     ): void => {
-      pluginEngine.invokeSingle('errorReportingProvider.notify', client, error, state, logger);
+      if (httpClient) {
+        const { component, tolerateNonErrors, errorFramesToSkip, normalizedError } =
+          getConfigForPayloadCreation(error, errorState?.severityReason.type as string);
+
+        // Generate the error payload
+        const errorPayload = ErrorFormat.create(
+          normalizedError,
+          tolerateNonErrors,
+          errorState as ErrorState,
+          component,
+          errorFramesToSkip,
+          logger,
+        );
+
+        // filter errors
+        if (!isRudderSDKError(errorPayload.errors[0])) {
+          return;
+        }
+
+        // enrich error payload
+        const bugsnagPayload = getBugsnagErrorEvent(errorPayload, errorState as ErrorState, state);
+
+        // send it to metrics service
+        httpClient?.getAsyncData({
+          url: `https://sdk-metrics.rudderstack.com/sdkmetrics`,
```

Comment:
Please fix this URL (`<dataPlaneUrl>/rsaMetrics` is now accessible for all the workspaces.)